From b99c6c50e79514f87a5fba716b9ca2ae21544dbd Mon Sep 17 00:00:00 2001
From: Ido Reis <idor@ti.com>
Date: Thu, 6 Oct 2011 19:30:33 +0200
Subject: [PATCH] wl12xx-tool.sh usage of busybox

change the script sh settings from #!/bin/sh to #!/data/busybox/sh

Signed-off-by: Ido Reis <idor@ti.com>
---
 scripts/wl12xx-tool.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/wl12xx-tool.sh b/scripts/wl12xx-tool.sh
index d2f2072..5f1cda7 100755
--- a/scripts/wl12xx-tool.sh
+++ b/scripts/wl12xx-tool.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/busybox/sh
 
 # PLT utility for TI's wireless driver wl12xx
 #
-- 
1.7.0.4

